cmd/bin.ts

```diff
@@ -31,13 +31,14 @@ yargs(hideBin(process.argv))
     "configure a camera",
     (yargs) => {
       return yargs
-        .option("debug", { default: false })
+        .option("debug", { describe: "Log debug messages", default: false })
         .option("ansi", { default: false })
         .boolean(["debug", "ansi"])
-        .option("discovery_ip", { default: "192.168.1.255" })
+        .option("discovery_ip", { describe: "UDP broadcast IP address", default: "192.168.1.255" })
```

Comment:
This address isn't necessarily broadcast - you _can_ send packets to a specific camera, mostly useful for debugging. Not sure if it's worth it to list that in the help though

Reply:
Updated the description and the README file.